dags/js/tasks/update-tiff-references-to-jpeg-in-article.js

```diff
@@ -0,0 +1,60 @@
+const libxmljs = require('libxmljs');
+const jatsXpaths = require('../xml/jats-xpaths');
+const namespaces = require('../xml/namespaces');
+const xpaths = require('../xml/xpaths');
+
+
+function updateTiffReferencesToJPEGInArticle(xmlStringBuffer) {
+
+  let xmlDoc = libxmljs.parseXml(xmlStringBuffer.toString());
+
+  xmlDoc.find(jatsXpaths.IMAGE_BY_TIFF_MIMETYPE).forEach((element) => {
+
+    for (let attribute of element.attrs()) {
+
+      try {
+        var prefix = attribute.namespace().prefix();
+      } catch(error) {
+        if (attribute.name() === 'mime-subtype') {
+          attribute.value('jpeg');
+        }
+      }
+
+      if (prefix === 'xlink' && attribute.name() === 'href') {
+        attribute.value(attribute.value().replace(new RegExp('\\.\\w+$'), '.jpg'));
+      }
+
+    }
+
+  });
+
+  xmlDoc.find(xpaths.XLINK_HREF_CONTAINS_TIF, {'xlink': namespaces.XLINK}).forEach((element) => {
+
+    for (let attribute of element.attrs()) {
+
+      try {
+        var prefix = attribute.namespace().prefix();
+      } catch(error) {
+        continue;
+      }
+
+      if (prefix === 'xlink' && attribute.name() === 'href') {
+        attribute.value(attribute.value().replace(new RegExp('\\.\\w+$'), '.jpg'));
+        break;
+      }
+
+    }
+
+  });
+
+  // adding false to document.toString should remove formatting but
+  // a few newlines (\n) can still be found
+  let xmlDocAsString = xmlDoc.toString(false);
+
+  // actually remove formatting
+  xmlDocAsString = xmlDocAsString.replace(new RegExp('\n', 'g'), '');
+
+  return Buffer.from(xmlDocAsString);
+}
+
+module.exports = updateTiffReferencesToJPEGInArticle;
```

Comment:
Technically it would make no difference but when you have many open at once that look similar, it can be a helpful reminder. Either way, would refactor in another PR.